test/upgrade/1.0.0-1.1.0.ts

```diff
@@ -0,0 +1,234 @@
+    tokenId = nftMintedEvent.args[1];
+
+    const minimalPrice = 1000n;
+    const customItemPrice = 10000n;
+    const selfSaleData = ethers.AbiCoder.defaultAbiCoder().encode(
+      ["uint256", "uint256"],
+      [minimalPrice, customItemPrice],
+    );
+
+    const selfSaleWrapType = 0;
+    await mockToken.approve(fermionProtocolAddress, minimalPrice * 5n);
+    await offerFacet.unwrapNFT(tokenId, selfSaleWrapType, selfSaleData);
+
+    // Calculate storage slots
+    const offerStorageSlot = getMappingStorageSlot(offerLookupsSlot, offerId);
+    offerItemQuantitySlot = BigInt(offerStorageSlot) + 10n;
+    offerFirstTokenIdSlot = BigInt(offerStorageSlot) + 11n;
+
+    tokenIdStorageSlot = BigInt(getMappingStorageSlot(tokenLookupsSlot, tokenId));
+    bosonProtocolFeeTokenLocation = tokenIdStorageSlot + 7n;
+    fermionFeeAmountTokenLocation = tokenIdStorageSlot + 8n;
+    verifierFeeTokenLocation = tokenIdStorageSlot + 9n;
+    facilitatorFeeAmountTokenLocation = tokenIdStorageSlot + 10n;
+  });
+
+  context("Backfilling", function () {
+    it("should backfill token fees and offer data during upgrade", async function () {
+      // make sure token fees are not 0
+      expect(await getStorageAt(fermionProtocolAddress, bosonProtocolFeeTokenLocation)).to.not.equal(0n);
+      expect(await getStorageAt(fermionProtocolAddress, fermionFeeAmountTokenLocation)).to.not.equal(0n);
+      expect(await getStorageAt(fermionProtocolAddress, verifierFeeTokenLocation)).to.not.equal(0n);
+      expect(await getStorageAt(fermionProtocolAddress, facilitatorFeeAmountTokenLocation)).to.not.equal(0n);
+
+      // set all fees to 0
+      await setStorageAt(fermionProtocolAddress, bosonProtocolFeeTokenLocation, BigInt(0));
+      await setStorageAt(fermionProtocolAddress, fermionFeeAmountTokenLocation, BigInt(0));
+      await setStorageAt(fermionProtocolAddress, verifierFeeTokenLocation, BigInt(0));
+      await setStorageAt(fermionProtocolAddress, facilitatorFeeAmountTokenLocation, BigInt(0));
+
+      // make sure item quantity and first token id are not 0
+      expect(await getStorageAt(fermionProtocolAddress, offerItemQuantitySlot)).to.not.equal(0n);
+      expect(await getStorageAt(fermionProtocolAddress, offerFirstTokenIdSlot)).to.not.equal(0n);
+
+      // set them to 0
+      await setStorageAt(fermionProtocolAddress, offerItemQuantitySlot, BigInt(0));
+      await setStorageAt(fermionProtocolAddress, offerFirstTokenIdSlot, BigInt(0));
+
+      const offerData = [
+        {
+          offerId: offerId,
+          itemQuantity: "1",
+          firstTokenId: tokenId,
+        },
+      ];
+
+      const backFillOfferCalldata = backfillingFacet.interface.encodeFunctionData("backFillOfferData", [offerData]);
+
+      const tokenFeesData = [
+        {
+          tokenId: tokenId,
+          bosonProtocolFee: 10,
+          fermionFeeAmount: 20,
+          verifierFee: 10,
```

Comment:
Maybe pick a different number that was the original `verifierFee`, just to make sure the new value is stored.

Or alternatively, after you set the fees to 0 with `setStorageAt` make another check with `getStorageAt` to see they really are 0.